larray/tests/test_la.py

```diff
@@ -145,16 +145,16 @@ def test_equals(self):
     def test_group(self):
```

Comment:
this test is now at least partially redundant with the test_getitem test below, both should be merged (or this one should disappear if it does not bring anything different).